tomcat/src/main/java/camp/nextstep/http/domain/HttpCookies.java

```diff
@@ -0,0 +1,64 @@
+package camp.nextstep.http.domain;
+
+import camp.nextstep.http.exception.InvalidHttpHeaderException;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class HttpCookies {
+    private static final String COOKIES_DELIMITER = ";";
+    private static final String COOKIE_DELIMITER = "=";
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final int SPLIT_LIMIT = 2;
+    private final Map<String, String> cookies;
```

Comment:
```suggestion
    private static final int SPLIT_LIMIT = 2;
    
    private final Map<String, String> cookies;
```

상수와 상태 값은 구분지어주면 좋습니다~